machine-readable-to-human-readable-date-time/.envrc

```diff
@@ -1,9 +1,2 @@
 #!/usr/bin/env bash
-# the shebang is ignored, but nice for editors
-
-if type -P lorri &>/dev/null; then
-  eval "$(lorri direnv)"
-else
-  echo 'while direnv evaluated .envrc, could not find the command "lorri" [https://github.com/nix-community/lorri]'
-  use nix
```

Comment:
Interesting that this already falls back to `use nix` if `lorri` is unavailable.

Reply:
Yeah, that's the default code generated by `lorri init`. But using nix-direnv results in simpler code *and* less to install.